staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -289,10 +289,8 @@ func (o *DrainCmdOptions) RunDrain() error {
 
 	for _, info := range o.nodeInfos {
 		var err error
-		if !o.drainer.DryRun {
-			err = o.deleteOrEvictPodsSimple(info)
-		}
-		if err == nil || o.drainer.DryRun {
+		err = o.deleteOrEvictPodsSimple(info)
+		if err == nil {
```

Comment:
By now you should be aware that people call me short-if :policeman: :wink: the above 3 lines can be easily squashed into a single:
```
if err := o.deleteOrEvictPodsSimple(info); err == nil {
```

Reply:
of course : ) thanks